### DISCUSSION TITLE
Add IIIF manifest as parameter to url

### DISCUSSION THREAD

**Discussion by andybuki** [jbaiter/pdiiif#14] [General]
Hi Johannes, I want to ask you about the following scenario:

I add to my application page a button that I call PDF generation.

![image](https://user-images.githubusercontent.com/4548460/210973239-b7c36f48-72e5-4cf1-a2d1-89bf880c6495.png)

I know the URL of the IIIF manifest, which I want to add at the next step to pdfiiif application.

Can you recommend how to integrate it into Manifest URL?

![image](https://user-images.githubusercontent.com/4548460/210974143-95f7b696-720d-42b9-8801-356b2016192f.png)

**Comment by jbaiter** [comment]
Excellent and (in hindsight obvious) idea, will implement asap!

**Comment by jbaiter** [comment]
Live at https://pdiiif.jbaiter.de now, use the request parameter `manifest`, e.g.:
https://pdiiif.jbaiter.de/?manifest=https://api.digitale-sammlungen.de/iiif/presentation/v2/bsb10752064/manifest

**Reply by andybuki** [reply]
perfect, thanks
